_includes/master/manifests/calico-config.yaml

```diff
@@ -0,0 +1,127 @@
+# This ConfigMap is used to configure a self-hosted {{include.prodname}} installation.
```

Comment:
How about a jekyll comment at the top of each of these include manifests that explains which `.include` flags it accepts. For this manifest, for example:

```
{% comment %}

calico-config.yaml acccepts the following include flags:

| Name        | Accepted Values |
|-------------|-----------------|
| datastore   | kdd, etcd       |
| typha       | true, false     |
| network     | calico, flannel |
| calico_ipam | true, false     |

{% endcomment %}
```